common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayCollection.scala

```diff
@@ -1,23 +1,57 @@
-package uk.ac.wellcome.display.models.v2
+package uk.ac.wellcome.display.models
 
-import io.circe.generic.extras.JsonKey
 import io.swagger.v3.oas.annotations.media.Schema
-import uk.ac.wellcome.models.work.internal.Collection
+
+import uk.ac.wellcome.models.work.internal._
+import uk.ac.wellcome.display.models.v2.DisplayWorkV2
 
 @Schema(
   name = "Collection",
-  description = "A collection that a work is part of"
+  description = "A hierarchical collection of works."
 )
 case class DisplayCollection(
-  @Schema(description = "The label of the collection") label: Option[String],
-  @Schema(description = "Where in the hierarchy a work is in the collection") path: String,
-  @JsonKey("type") @Schema(name = "type") ontologyType: String = "Collection"
+  @Schema(
+    description =
+      "A slash separated path containing the position within the hierarchy."
+  ) path: String,
+  @Schema(
+    description =
+      "The level of the node. Either Collection, Section, Series, SubSeries or Item"
```

Comment:
Maybe not worth listing these out as they're already out of date.

Reply:
:disappointed: